charts/seed-monitoring/charts/grafana/dashboards/owners/node-details-dashboard.json

```diff
@@ -107,10 +107,10 @@
         "showThresholdMarkers": true,
         "text": {}
       },
-      "pluginVersion": "7.5.4",
+      "pluginVersion": "7.5.16",
```

Comment:
Was the update of the plugin version intentional?

Reply:
The PR contains 2 commits: the "uninteresting" change of the plugin version and a commit that contains the promql queries.
If a dashboard was created with an older version of Grafana that is used to export it, the dashboard json can contain "uninteresting" changes that are still worth versioning, otherwise we would always have to unstage those changes.